app/javascript/components/pages/ApartmentIndex.js

```diff
@@ -2,8 +2,12 @@ import React, {Component} from 'react'
 
 class ApartmentIndex extends Component {
   render() {
+    // const {apartments} = this.props
     return(
-      <h3>Apartment Index</h3>
+      <div className="page-body">
```

Comment:
I may be missing it but it doesn't look like you have a page-body class to call upon. You may want to develop page body before you ship it.